app/src/main/java/net/squanchy/schedule/SchedulePageView.kt

```diff
@@ -25,9 +25,9 @@ import net.squanchy.support.unwrapToActivityContext
 import timber.log.Timber
 
 class SchedulePageView @JvmOverloads constructor(
-    context: Context,
-    attrs: AttributeSet,
-    defStyleAttr: Int = 0
+        context: Context,
```

Comment:
I'm quite sure your autoformatter is set incorrectly, the continuation indent is 4 not 8 ❌ 

![image](https://user-images.githubusercontent.com/153802/37955817-8b976010-31a1-11e8-9ccf-d3e8fab44dd7.png)

Reply:
I'll double check today, but I did change the scheme to project before reformatting

Reply:
Did you use Android Studio 3.0.1? When we updated the AGP to 3.1 we also needed to change the code styling location as it's been updated in the new IntelliJ, so if you're using an old version of AS it would be incorrectly detected

Reply:
I see, yeah I'll have to wait for ASide 3.1 to be stable 👅

Reply:
which seems to be today! Updating

Reply:
WAITING YOU SAY? https://android-developers.googleblog.com/2018/03/android-studio-3-1.html ahaha

Reply:
By the way KtLint should reformat things correctly

Reply:
cf743d3